lib/Onnxifi/onnxifiGlow.cpp

```diff
@@ -260,9 +260,9 @@ onnxReleaseEvent(onnxEvent event) {
 
 /// Parse an ONNXIFI graph and convert it for a particular backend.
 ONNXIFI_PUBLIC ONNXIFI_CHECK_RESULT onnxStatus ONNXIFI_ABI onnxInitGraph(
-    onnxBackend backend, size_t onnxModelSize, const void *onnxModel,
-    uint32_t weightsCount, const onnxTensorDescriptorV1 *weightDescriptors,
-    onnxGraph *graph) {
+    onnxBackend backend, const uint64_t *auxPropertiesList,
```

Comment:
I don't find where auxPropertiesList is used in this onnxInitGraph. Is it some special param?

Reply:
it's not used ATM. But the interface itself has changed, see onnxifi.h in onnx repo.